android-app/app/src/main/java/de/dhbw/ka/se/fibo/ApplicationState.java

```diff
@@ -0,0 +1,51 @@
+package de.dhbw.ka.se.fibo;
+
+import android.content.Context;
+import android.util.Log;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.time.chrono.ChronoLocalDate;
+import java.time.chrono.ChronoLocalDateTime;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import de.dhbw.ka.se.fibo.models.Cashflow;
+import de.dhbw.ka.se.fibo.models.CashflowType;
+import de.dhbw.ka.se.fibo.models.Category;
+
+public class ApplicationState {
+
+    private final Context context;
+    private static ApplicationState instance;
```

Comment:
if im informed correctly there is no memory leak because of the usage of instances. maybe supress the warning with this hint?